openshift/topics/tutorials.adoc

```diff
@@ -544,33 +497,6 @@ deploymentconfig "sso" scaled
 === Example Workflow: Migrating The {project_name} Server's Database Across Environments
 This tutorial focuses on migrating the Red Hat Single Sign-On server database from one environment to another or migrating to a different database. It assumes steps described in xref:Preparing-SSO-Authentication-for-OpenShift-Deployment[Preparing {project_name} Authentication for OpenShift Deployment] section have been performed already.
 
-==== Deploying the {project_name} MySQL Application Template
```

Comment:
Would keep this section, but replace it with PostgreSQL template case. Looks that deploying `Deploying the {project_name} MySQL Application Template` is one step from the tutorial. This it's preferred to keep it here, but point to postgresql example case.